SimeonInstaller.ps1

```diff
@@ -1657,6 +1769,358 @@ CRLFOption=CRLFAlways
+                # Repositories > rename $Project to default
+                Write-Information "Renaming the repository: $Project to default"
+                $repos = (Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://dev.azure.com/$Organization/$projectId/_apis/git/repositories?api-version=6.0" -Method Get }).value
+                if ($repos.name -contains "$Project") {
+                    $repoId = ($repos |? { $_.name -eq "$Project" }).id
+                    Invoke-WithRetry { Invoke-RestMethod  -Headers $AuthenicationHeader -Uri "https://dev.azure.com/$Organization/$projectId/_apis/git/repositories/$repoId`?api-version=5.0" -Method Patch -Body '{"name":"Default"}' -ContentType "application/json" } | Out-Null
+                }
+
+                # Overview > uncheck Boards and Test Plans
+                Write-Information "Updating project settings turning off Boards and Test Plans"
+                Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://dev.azure.com/$Organization/_apis/FeatureManagement/FeatureStates/host/project/$projectId/ms.vss-work.agile?api-version=4.1-preview.1" -Method Patch -ContentType "application/json" -Body @"
+                    {
+                        "featureId": "ms.vss-work.agile",
+                        "scope": {
+                            "settingScope": "project",
+                            "userScoped": false
+                        },
+                        "state": 0
+                    }
+"@
+                } | Out-Null
+
+                # Overview > uncheck Artifacts
+                Write-Information "Updating project settings turning off Artifacts"
+                Invoke-WithRetry { Invoke-RestMethod -Header $AuthenicationHeader -Uri "https://dev.azure.com/$Organization/_apis/FeatureManagement/FeatureStates/host/project/$projectId/ms.feed.feed?api-version=4.1-preview.1" -Method Patch -ContentType "application/json" -Body @"
+                    {
+                        "featureId": "ms.feed.feed",
+                        "scope": {
+                            "settingScope": "project",
+                            "userScoped": false
+                        },
+                        "state": 0
+                    }
+"@
+                } | Out-Null
+            }
+        }
+
+        ## Project settings
+        Write-Information "Configuring pipeline settings"
+        Invoke-Command -ScriptBlock {
+            # Pipelines > Settings > uncheck Limit job authorization scope to current project for non-release pipelines (enforceReferencedRepoScopedToken), Limit job authorization scope to referenced Azure DevOps repositories (enforceJobAuthScope), and Limit variables that can be set at queue time (enforceSettableVar)
+            Write-Information "Limiting job authorization scope to current project for non-release pipelines"
```

Comment:
aren't you doing the OPPOSITE? "uncheck Limit job authorization scope to current project for non-release pipelines "